src/main/java/com/sarality/validation/Validators.java

```diff
@@ -48,22 +49,26 @@ private Validators register(String fieldName, Validator validator, ErrorMessage
     return this;
   }
 
-  public boolean validate(Activity activity, FormData formFieldData) {
+  public boolean validate(Activity activity, View contextView, FormData formFieldData) {
     boolean isValidForm = true;
     for (String fieldName : validatorListMap.keySet()) {
-      boolean isValidField = validateField(activity, formFieldData, fieldName);
+      boolean isValidField = validateField(activity, contextView, formFieldData, fieldName);
       isValidForm = isValidForm & isValidField;
     }
     return isValidForm;
   }
 
-  public boolean processErrors(Activity activity, List<ErrorCode> errorCodes) {
+  public boolean validate(Activity activity, FormData formFieldData) {
```

Comment:
nope, if that had been the case I would definitely have paused. This was easily handled within validators itself. Good job you!